### ISSUE TITLE
Start of next day is earlier than in the Settings in 2.9.2beta4

### ISSUE THREAD

**Issue by homocomputeris** [ankidroid/Anki-Android#5637]
###### Reproduction Steps
1. Set up 'Start of next day' in Settings to, say, 4am

###### Expected Result
Start new day at 4am

###### Actual Result
Next day starts before that time; it seems like 1 h before

###### Debug info
AnkiDroid Version = 2.9.2beta4
Android Version = 9
ACRA UUID = 1e82c68a-fdc6-4201-ace5-5f2d2d4c6039

###### Research
*Enter an [ x ] character to confirm the points below:*

- [x] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid
- [x] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue
- [x] I have searched for similar existing issues here and on the user forum

**Comment by mikehardy** [comment]
Interesting - this is also coincidentally the time when daylight savings switches for people and we inevitably get complaints about sync errors from whatever countries switched their savings times this year without android vendors patching the system in time. I wonder if a similar thing is happening here where start of next day is offset from UTC and it's having a DST error?

**Comment by HairyPhil** [comment]
Yes, it has always ignored daylight savings time. In summer, if I want AnkiDroid to reset at 00:00, I have to set it to reset at 01:00.

**Comment by mikehardy** [comment]
I'm not sure if this is something we should handle or not - I suppose we should - but that reports makes sense based on my understanding the Anki ecosystem runs internally (read as: all dates are stored as) UTC with no offset I think

**Comment by HairyPhil** [comment]
It's a problem if you travel to several different time zones. It should reset at local time. I shouldn't have to change my AnkiDroid settings twice a year plus every time I get on a plane. I can't think of any other product that would make me do that. Android certainly has no trouble knowing what the local time is.

I know the reasoning is that if it used local time then there are days that would be longer or shorter than 24 hours, which in theory could slightly mess up review scheduling. But better that than have it unexpectedly reset on me at 8PM when I'm halfway through a session.

**Comment by mikehardy** [comment]
not disagreeing, but in reply to "I can't think of any other product..." I can't really either, but then again I can't actually think of any other app I use that is actually open source. Most of the polish issues like this are because in a volunteer system polish is usually not valued vs stability and new features and ability to customize. Fact of volunteer self-interest unfortunately

**Comment by david-allison-1** [comment]
stalebot was a little too hasty

**Comment by HairyPhil** [comment]
Bot, seriously? Now both tickets for this issue are closed. One needs to stay open.

**Comment by boolbag** [comment]
Howdy all, 
some screenshots to illustrate the issue as of 2.15 alpha 21.
Not a big deal, but it's puzzled me for a while. :-)
Wishing y'all a great day.

![Anki-at-3-am](https://user-images.githubusercontent.com/7551671/104620744-185be900-566e-11eb-8eef-40cca070ed50.jpg)

**Comment by dae** [comment]
Should be addressed by the new backend.
